packages/contractkit/src/kit.ts

```diff
@@ -239,4 +239,17 @@ export class ContractKit {
       ...tx,
     }
   }
+
+  async getCurrentEpoch(): Promise<number> {
+    const validators = await this.contracts.getValidators()
+    const epochSize = await validators.getEpochSize()
+    const currentBlock = await this.web3.eth.getBlockNumber()
+    return Math.floor(currentBlock / epochSize)
+  }
+
+  async epochToBlockNumber(epochNumber: number): Promise<number> {
```

Comment:
what about renaming this to `getLastBlockNumberForEpoch` or something similar... since it's not clear which block number for an epoch this fn will return